docs/modules/user-guide/partials/con_devfile-registry.adoc

```diff
@@ -0,0 +1,48 @@
+[id="devfile-registry_{context}"]
+= Devfile registry
+
+[role="_abstract"]
+A devfile registry is a service that stores and provides devfile stacks to Kubernetes developer tools like odo, Eclipse Che, and the OpenShift Developer Console. Therefore, you can access devfiles through the devfile registry.  Devfile registries are based on the Oracle Cloud Infrastructure (OCI) Specification, and devfile stacks are stored as OCI-artifacts in the registry.
+
+Each devfile stack corresponds to a specific runtime or framework, such as Node.js, Quarkus, or WildFly. A devfile stack also contains resources like a `devfile.yaml` file, logo, and outer loop resources. Outer loop runs on containers and entails code reviews and integration tests, typically automated by continuous integration/continuous delivery (CI/CD) pipelines. These devfile stacks provide developers with templates to get started developing cloud-native applications.
+
+== Devfile registry components
+
+A devfile registry has two components:
+
+ * Devfile index server
+ * OCI registry server
+
+.Devfile index container
+
+The devfile index container image does the following actions:
+
+* Uses the `index.json` file to obtain metadata about the stacks and samples and hosts this metadata in the registry for tools to consume.
+* Bootstraps the OCI registry with the devfile stacks.
+* Provides an API to interact with the OCI registry and to retrieve devfile stacks.
+
+.OCI registry
+
+The devfile stacks in a devfile registry are stored in an OCI-compatible artifact registry. The artifact registry is based on the reference implementation of an OCI registry.
+
+When a devfile registry is deployed, the stacks belonging to the devfile registry are pushed to the OCI registry as multi-layer artifacts.
```

Comment:
From previous issues, I know we were concerned with inappropriately calling our users directly. But we have a lot of instances of passive voice throughout this doc, which is not advised by the style guide. 

@rkratky how concerned do you think we should be about passive voice in this doc before we merge? 

If we are fine with using second person, I can quickly make changes to active voice throughout. Like in this instance, I could write, "When you deploy a devfile registry, the stacks are pushed..." 

Otherwise, we can say something like, "When users deploy a devfile registry, the stacks are pushed..." 

Basically, I wanted to get your opinion before making so many changes to the verb tense throughout, thanks!

Reply:
@jc-berger, I don't have a problem with passive voice here. It's a concept doc, and it only provides a background explanation. It's not a procedure that would instruct users to do things. Forcing it wouldn't be helpful. E.g.:

_"When users deploy a devfile registry, the stacks are pushed..."_

That would be weird. It's usually not _users_ who deploy the registry. It's the admin, or maybe the service provider. We want the docs to be authoritative (_do this, do that_ instead of _consider blah blah_), but only when makes sense.